src/protocol.ts

```diff
@@ -56,7 +57,7 @@ interface API {
 	deploy(
 		name: string,
 		env: string[],
-		plan: string,
+		plan: Plans,
```

Comment:
look at Line 192.

Reply:
Sorry for making such a mistake I have solve it and waiting to be reviewed